cli/src/command.rs

```diff
@@ -27,6 +27,9 @@ use sp_core::crypto::Ss58AddressFormatRegistry;
 use sp_keyring::Sr25519Keyring;
 use std::net::ToSocketAddrs;
 
+#[cfg(feature = "try-runtime")]
+use try_runtime_cli::block_building_info::timestamp_with_babe_info;
```

Comment:
I would move this to line 641 inside the feature-gated scope. There is already a few imports there.

Reply:
sure, moved